.travis.yml

```diff
@@ -23,7 +24,7 @@ before_install:
   - cd WcaOnRails/
 before_script:
   - bundle exec rake db:reset RAILS_ENV=test
-sudo: false
+sudo: required
```

Comment:
It's a shame that we'll have to lose Travis's container-based infrastructure here (see [this doc](https://docs.travis-ci.com/user/migrating-from-legacy/#Why-migrate-to-container-based-infrastructure%3F) explaining why it's nicer than their legacy vm infrastructure).

It looks like it's possible for us to opt-in to Ubuntu 14.04 (Trusty): https://docs.travis-ci.com/user/trusty-ci-environment/.  That version of Ubuntu might have a new enough version of openssl installed that things will Just Work, and would let us stick with the container environment. Could you give that a shot?